server/users.go

```diff
@@ -64,10 +65,10 @@ func (r *userRequest) ValidRoles() error {
 			}
 			orgs[r.Organization] = true
 			switch r.Name {
-			case MemberRoleName, ViewerRoleName, EditorRoleName, AdminRoleName:
+			case roles.MemberRoleName, roles.ViewerRoleName, roles.EditorRoleName, roles.AdminRoleName:
 				continue
 			default:
-				return fmt.Errorf("Unknown role %s. Valid roles are 'viewer', 'editor', 'admin', and 'superadmin'", r.Name)
+				return fmt.Errorf("Unknown role %s. Valid roles are 'member', 'viewer', 'editor', 'admin', and 'superadmin'", r.Name)
```

Comment:
not sure superadmin is exactly a valid role. but this comes back to that same strangeness as above: superadmin is more of a status than a role.

Reply:
fixed